mysite/w6/beanstalk.py

```diff
@@ -1,140 +1,146 @@
+        total_owned = gold_foods[gold_food_code]
+
+        # remove 10k from the amount of gold food owned if the first 10k hasn't been deposited yet
+        adjusted_total_owned = total_owned if session_data.account.sneaking['Beanstalk'][gold_food_code]['Beanstacked'] else total_owned - BEANSTACK_GOAL
+
+        # mark progress as 0% if less than 10k is owned
+        super_beanstack_progress[gold_food_code] = max(0, adjusted_total_owned)
+
+    foods_to_super_beanstack = []  # If food is not already Beanstacked and player has less than required amount (10k)
+    for foodName, foodValuesDict in session_data.account.sneaking['Beanstalk'].items():
+        if not foodValuesDict['SuperBeanstacked']:
+            if super_beanstack_progress[foodName] < SUPER_BEANSTACK_GOAL:
+                foods_to_super_beanstack.append(foodName)
+            else:
+                foods_ready_to_deposit.append(f"{getItemDisplayName(foodName)}: 100k Super Beanstack")
+
+    foods_finished = sum([v['Beanstacked'] for v in session_data.account.sneaking['Beanstalk'].values()])
     tier = f"{foods_finished}/{len(gold_foods)*2}"
 
     advice_deposit = [
         Advice(
-            label=getItemDisplayName(codename),
-            picture_class=getItemDisplayName(codename),
+            label=foodname,
+            picture_class=foodname.split(":")[0],
         )
-        for codename in foods_to_deposit
+        for foodname in foods_ready_to_deposit
     ]
 
-    advice_10k = [
+    advice_beanstack = [
         Advice(
             label=getItemDisplayName(codename),
             picture_class=getItemDisplayName(codename),
-            progression=f"{int(gold_foods[codename]) / 10**4:.02%}",
+            progression=f"{int(gold_foods[codename]) / BEANSTACK_GOAL:.02%}",
         )
-        for codename in foods_to_10k
+        for codename in foods_to_beanstack
     ]
 
-    advice_100k = [
+    advice_super_beanstack = [
         Advice(
             label=getItemDisplayName(codename),
             picture_class=getItemDisplayName(codename),
-            progression=f"{int(gold_foods[codename]) / 10**5:.02%}",
+            progression=f"{int(super_beanstack_progress[codename]) / SUPER_BEANSTACK_GOAL:.02%}",
         )
-        for codename in foods_to_100k
+        for codename in foods_to_super_beanstack
     ]
 
     group_upgrade = AdviceGroup(
         tier="",
         pre_string="Upgrade the Beanstalk to upgrade foods further",
         advices=[
             Advice(
-                label="Buy \"Supersized Gold Beanstacking\" from the Jade Emporium",
-                picture_class="supersized-gold-beanstacking"
+                label='Buy "Supersized Gold Beanstacking" from the Jade Emporium',
+                picture_class="supersized-gold-beanstacking",
             )
-        ]
+        ],
     )
 
     group_deposit = AdviceGroup(
         tier="",
-        pre_string="Deposit these golden foods",
+        pre_string="Golden Foods ready for deposit",
```

Comment:
nit: Figured I might as well bring it up but we use "Golden Foods" and "golden foods", can this "F" be "f" instead?

Reply:
I prefer Golden Food as it feels like a proper name. Same way I'd type Mountain Bread, not Mountain bread.